packages/litexa/src/parser/litexa.pegjs

```diff
@@ -1539,11 +1539,10 @@ SlotTypeStatement
   }
 
 /* litexa [otherwise]
-Defines a catch all intent that will be executed should
-its parent state receive an intent that it has no explicit
-handler for. It's always possible for the user to say
-something unexpected, so you should generally always have
-something to say to guide them back on track in these cases.
+Defines a catch all intent that will be executed should its parent state receive
```

Comment:
catch all intent -> catch-all handler

Reply:
Polished formulation a bit - let me know if that makes sense.